src/index.js

```diff
@@ -6,3 +6,5 @@ if (!window.customElements.get('clipboard-copy')) {
   window.ClipboardCopyElement = ClipboardCopyElement
   window.customElements.define('clipboard-copy', ClipboardCopyElement)
 }
+
+export {copyInput, copyNode, copyText} from './clipboard'
```

Comment:
These functions are currently implementation details with backwards compatibility for browsers without the clipboard API. Soon we'll be able to just say `navigator.clipboard.writeText('hello')` in application code.

So is it worth supporting these as public API in a 1.0 release of the custom element? It might be more valuable to polyfill `clipboard.writeText` in the application.